aspnetcore/security/key-vault-configuration.md

```diff
@@ -210,402 +210,9 @@ config.AddAzureKeyVault(
 
 | Property         | Description |
 | ---------------- | ----------- |
-| `Client`         | <xref:Microsoft.Azure.KeyVault.KeyVaultClient> to use for retrieving values. |
-| `Manager`        | <xref:Microsoft.Extensions.Configuration.AzureKeyVault.IKeyVaultSecretManager> instance used to control secret loading. |
+| `Client`         | <xref:Azure.Security.KeyVault.Secrets.SecretClient> to use for retrieving values. |
+| `Manager`        | <xref:Azure.Extensions.Aspnetcore.Configuration.Secrets.KeyVaultSecretManager> instance used to control secret loading. |
 | `ReloadInterval` | `Timespan` to wait between attempts at polling the key vault for changes. The default value is `null` (configuration isn't reloaded). |
-| `Vault`          | Key vault URI. |
-
-## Use a key name prefix
```

Comment:
Remove large section of the article is because most of them were duplicate. 
"Use a key name prefix" was removed because AddAzureKeyVault does not accept implementation of IKeyVaultSecretManager any more, this is no longer needed.

Reply:
The KeyVaultSecretManager provides an almost identical interface to the IKeyVaultSecretManager. This section should be updated to use that instead of being removed.

Reply:
@pakrym, "Use a key name prefix" added back and updated.